header.php

```diff
@@ -1,385 +1,385 @@
+				.starRemoveButton {
+					opacity: 1;
+				}
+			}
+			
+			.identifier:hover ~ .starRemoveButton:not(.disabled), .starRemoveButton:hover:not(.disabled) {
+				opacity: 0.5;
+				cursor: pointer;
+			}
+			
+			.disabled {
+				pointer-events: none;
+			}
+			
+			.star-value {
+				font-size: 18px;
+			}
+			
+			.star {
+				white-space: nowrap;
+			}
+			
+			span.subText{
+				font-size: 11px;
+				color: #a8a8a8;
+			}
+			
+			.ratingContainer {
+				background-color: DarkSlateGrey;
+				align-items: center;
+			}
+			
+			.flex-container {
+				display: flex;
+				width: 100%;
+			}
+			
+			.flex-child {
+				margin: 0.25em;
+				vertical-align: middle;
+			}
+				
+			.diffThumb {
+				height: 32px;
+				width: 32px;
+				border: 1px solid #ddd;
+				object-fit: cover;
+			}
+			
+			select {
+				margin: 0.2em;
+				padding: 0.2em;
+				border: none;
+				border-radius: 1px;
+				background-color: DarkSlateGrey;
+				color: white;
+			}
+			
+			.commentContainer {
+				background-color: DarkSlateGrey;
+				padding: 0px;
+				align-items: center;
+				align-content: center; /* new */
+				flex-wrap: wrap;
+			}
+			
+			.commentHeader {
+				align-items: center;
+				width: 100%;
+				background-color: #203838;
+				padding: 0.25em;
+				margin: 0;
+			}
+			
+			.comment {
+				padding-left: 1em;
+				padding-right: 1em;
+				margin-bottom: 1.5em;
+				min-height: 3em;
+				width: 100%;
+			}
+			
+			.commentComposer {
+				width: 100%;
+				text-align: right;
+				margin: 1em;
+			}
+			
+			textarea#commentForm {
+				width: 100%;
+				text-align: left;
+				resize: none;
+				height: 6em;
+				background-color: #203838;
+				color: white;
+				border: 1px solid white;
+			}
+			
+			input#commentSubmit {
+				margin: 1em;
+				width: 6em;
+				border: 1px solid white;
+				background-color: #203838;
+				color: white;
+			}
+			
+			input#commentSubmit:hover {
+				background-color: #182828;
+			}
+			
+			.icon-remove {
+				cursor: pointer;
+			}
+			
+			.osuTimestamp {
+				background-color: #203838;
+				border-radius: 2px;
+				padding: 0.05em;
+			}
+		</style>
+		<script>
+			function showResult(str) {
+			  if (str.length==0) {
+				document.getElementById("topBarSearchResults").innerHTML="";
+				document.getElementById("topBarSearchResults").style.display="none";
+				return;
+			  }
+			  var xmlhttp=new XMLHttpRequest();
+			  xmlhttp.onreadystatechange=function() {
+				if (this.readyState==4 && this.status==200) {
+				  document.getElementById("topBarSearchResults").innerHTML=this.responseText;
+				  document.getElementById("topBarSearchResults").style.display="block";
+				}
+			  }
+			  xmlhttp.open("GET","/beatmapSearch.php?q="+str,true);
+			  xmlhttp.send();
+			}
+			
+			function searchFocus() {
+				document.getElementById("topBarSearchResults").style.display="block";
+			}
+		</script>
+	</head>
+	<body>
+		<div class="topBar">
+			<a href="/" style="margin-right: 8px;">OMDB - osu! map database</a>
```

Comment:
not sure how you feel about this one but i think having some margin on this (and also the search bar) makes it look a little nicer, up to u, if wanted it can be removed ofc